WORKSPACE

```diff
@@ -16,7 +16,7 @@ csharp_repositories(use_local_mono = True)
 
 git_repository(
     name = "org_pubref_rules_node",
-    commit = "d93a80ac4920c52da8adccbca66a3118a27018fd",  # Oct 2, 2016
+    commit = "1c60708c599e6ebd5213f0987207a1d854f13e23",  # Mar 12, 2018
```

Comment:
Fixes this error:

```
ERROR: /tmpdir/external/org_pubref_rules_node/node/internal/node_binary.bzl:58:23: name 'set' is not defined
```

This is the master of pubref/rules_node as of this writing.